wire-grpc-tests/src/test/java/com/squareup/wire/GrpcClientTest.kt

```diff
@@ -1494,6 +1494,8 @@ class GrpcClientTest {
     assertThat(responseChannel.read()).isEqualTo(Feature(name = "house"))
     assertThat(responseChannel.read()).isNull()
     assertThat(grpcCall.responseMetadata!!["response-lucky-animal"]).isEqualTo("horse")
+
+    mockService.awaitSuccessBlocking()
```

Comment:
Should we move this to a before each rule? it seems is not being evaluated on all test and I'm not sure if that is intentional or not.